client/go.mod

```diff
@@ -1,12 +1,10 @@
 module github.com/kubernetes-csi/external-snapshotter/client/v4
 
-go 1.15
+go 1.16
 
 require (
-	golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d // indirect
-	golang.org/x/time v0.0.0-20200416051211-89c76fbcd5d1 // indirect
-	k8s.io/api v0.19.0
-	k8s.io/apimachinery v0.19.0
-	k8s.io/client-go v0.19.0
-	k8s.io/code-generator v0.19.0
+	k8s.io/api v0.21.0
```

Comment:
Don't update k8s to 0.21 as controller-gen has a dependency on the older k8s version.